aten/src/ATen/native/cpu/BinaryOpsKernel.cpp

```diff
@@ -103,6 +103,31 @@ void logical_xor_kernel(TensorIterator& iter) {
     });
 }
 
+void lt_kernel(TensorIterator& iter) {
+  if (iter.dtype() == ScalarType::Bool) {
+    AT_DISPATCH_ALL_TYPES_AND2(kBool, kBFloat16, iter.input_dtype(), "lt_cpu", [&]() {
+      cpu_kernel(iter,
+       [=](scalar_t a, scalar_t b) -> bool {
+         return a < b;
+       });
+    });
+  } else if (iter.dtype() == ScalarType::Byte) {
```

Comment:
We can delete the uint8 case now, right? In PyTorch 1.2, passing a uint8 `x` as the `out` tensor gives:
```
[ins] In [6]: torch.lt(y, 0, out=x)
../aten/src/ATen/native/LegacyDefinitions.cpp:110: UserWarning: torch.lt received 'out' parameter
with dtype torch.uint8, this behavior is now deprecated,please use 'out' parameter with dtype torc
h.bool instead.
```
It sounds like our long term plan is to add it back in the future to support arbitrary dtype for the `out` tensor, but it would be good to be consistent with the current PyTorch API while we don't have that.

Reply:
Changed to Bool